### ISSUE TITLE
Data retained after erasing browser cache

### ISSUE THREAD

**Issue by Gitoffthelawn** [mozilla-mobile/focus-android#2951]
**Steps to reproduce:**

1. Install Firefox Focus/Klar v2.5 for Android
2. Go to http://lucb1e.com/rp/cookielesscookies/
3. Enter a string in the input field and press 'Store'
4. Press Firefox Focus/Klar's Trash Can icon to clear the cache
5. Repeat step 2

**Actual results:**

After Step 5, Notice that the counter increases and the string is not deleted.

Repeating Step 4, or reloading the page will reset the counter and delete the string.

**Expected results:**

The counter should be reset and the string stored in Step 3 should be deleted.

See also:
https://bugzilla.mozilla.org/show_bug.cgi?id=906448
https://bugzilla.mozilla.org/show_bug.cgi?id=1474235

**Comment by ekager** [comment]
Hi @Gitoffthelawn , what do you mean Focus v2.5 ? We are currently on Release 6.0. Also, are you talking about WebView or GeckoView builds?

**Comment by Gitoffthelawn** [comment]
Hi @ekager. :)

I do mean Focus v2.5.  I haven't used Focus for quite a few months, and was catching up on some backlog when I submitted the issue.

IIRC, I had downloaded Focus from the Google Play Store.  So whatever was on the Play Store will determine whether it was WebView or GeckoView.

In any case, congrats on reaching 6.0 already.  The improvements must be impressive.  When I have some time (and a working Android device), I'll have to play around with Focus again.

What you (or someone else there) might want to do is to try the steps I outlined above, and see if you still get the same results.

**Comment by lobontiumira** [comment]
This issue still reproduces on both RC 93.1.1 build, and the latest debug build from main from 10/5 using a Google Pixel (Android 10).
After deleting the history, re-opening the page displays an increased number of visits, and the stored characters.

**Comment by lobontiumira** [comment]
Still reproducible on 12/7 Focus Nightly build with Samsung Galaxy Note 8 (Android 9).

**Comment by ionutbedregeanu** [comment]
GeckoView bug filled https://bugzilla.mozilla.org/show_bug.cgi?id=1773288

**Comment by ionutbedregeanu** [comment]
Close this ticket in favor of GV ticket https://bugzilla.mozilla.org/show_bug.cgi?id=1773288

**Comment by Gitoffthelawn** [comment]
Hopefully this will be prioritized to be fixed.  I reported this bug over 4 years ago.

Sadly, I had to give up waiting for this to be fixed several years ago, and found an alternative application that does not have this bug.

As a piece of (hopefully) constructive feedback: when bugs take years to fix (or even get assigned to anyone to start fixing them) it disincentivizes people from filing bug reports.  I have quite a few bugs for regular Firefox that have remained unresolved for over 2 decades.  I rarely choose to spend my time filing bug reports for Firefox these days as a result of these types of experiences.  I don't mean any offense in any way; I simply find it a better use of my time to file bug reports for projects that are managed in a more responsive way (see my GitHub and SourceForge histories for many examples).  I continue to support Mozilla, but I simply can't justify spending my time writing bug reports when a developer isn't even assigned to fix them.

I hope Mozilla can understand what I'm conveying and improve their processes.  Feel free to send this note to the CEO and upper management.  I wrote to the CEO over a year ago, but I never heard back.
